src/Sylius/Bundle/AdminBundle/Resources/views/Product/_showInShopUrl.html.twig

```diff
@@ -0,0 +1,13 @@
+{% if product.slug is empty %}
+    <div class="item" title="{{ 'sylius.ui.no_slug_for_channel'|trans }}">
+        <i class="remove icon"></i>
+        {{ 'sylius.ui.no_url_in'|trans }} {{ channel.name }}
+    </div>
+{% else %}
+    {% set url = sylius_channel_url(path('sylius_shop_product_show', {'slug': product.slug, '_locale': channel.defaultLocale.code}), channel) %}
```

Comment:
Should it use the slug from the translation corresponding to the channel's default locale?
How is this thing even working right now?